qcfractal/queue_handlers/queue_handlers.py

```diff
@@ -341,7 +309,7 @@ def build_queue(queue_type, queue_socket, db_socket, logger=None, **kwargs):
 
         adapter = dask_handler.DaskAdapter(queue_socket)
 
-    elif queue_type == "fireworks":
+    elif queue_type == "fireworks.core.launchpad.LaunchPad":
```

Comment:
Does it have to be that exact? Would a `elif 'fireworks' in queue_type.lower():` be safer or is that too vague?

Reply:
`queue_type` is now derived from object type so it does need to be exact. Using a string rather than `isinstance` so that we do not have to import Fireworks before evaluating the conditional.